cli/tests/test_help_command.rs

```diff
@@ -84,3 +79,145 @@ fn test_help() {
+    insta::assert_snapshot!(help_cmd_stdout, @r#"
+    Jujutsu (An experimental VCS)
+
+    To get started, see the tutorial at https://martinvonz.github.io/jj/latest/tutorial/.
+
+    Usage: jj [OPTIONS] <COMMAND>
+
+    Commands:
+      abandon           Abandon a revision
+      backout           Apply the reverse of a revision on top of another revision
+      bookmark          Manage bookmarks
+      commit            Update the description and create a new change on top [aliases: ci]
+      config            Manage config options
+      describe          Update the change description or other metadata [aliases: desc]
+      diff              Compare file contents between two revisions
+      diffedit          Touch up the content changes in a revision with a diff editor
+      duplicate         Create a new change with the same content as an existing one
+      edit              Sets the specified revision as the working-copy revision
+      evolog            Show how a change has evolved over time [aliases: evolution-log]
+      file              File operations
+      fix               Update files with formatting fixes or other changes
+      git               Commands for working with Git remotes and the underlying Git repo
+      help              Print this message or the help of the given subcommand(s)
+      init              Create a new repo in the given directory
+      interdiff         Compare the changes of two commits
+      log               Show revision history
+      new               Create a new, empty change and (by default) edit it in the working copy
+      next              Move the working-copy commit to the child revision
+      operation         Commands for working with the operation log [aliases: op]
+      parallelize       Parallelize revisions by making them siblings
+      prev              Change the working copy revision relative to the parent revision
+      rebase            Move revisions to different parent(s)
+      resolve           Resolve a conflicted file with an external merge tool
+      restore           Restore paths from another revision
+      root              Show the current workspace root directory
+      show              Show commit description and changes in a revision
+      simplify-parents  Simplify parent edges for the specified revision(s)
+      sparse            Manage which paths from the working-copy commit are present in the working copy
+      split             Split a revision in two
+      squash            Move changes from a revision into another revision
+      status            Show high-level repo status [aliases: st]
+      tag               Manage tags
+      util              Infrequently used commands such as for generating shell completions
+      undo              Undo an operation (shortcut for `jj op undo`)
+      version           Display version information
+      workspace         Commands for working with workspaces
+        
```

Comment:
nit: is it intentional that we have two blank lines here?

Reply:
Oh, actually no. It seems that `{subcommands}` comes with a `\n` at the end